src/box/relay.cc

```diff
@@ -544,10 +539,10 @@ relay_final_join(struct iostream *io, uint64_t sync,
 
 	/*
 	 * Save the first vclock as 'received'. Because firstly, it was really
-	 * received. Secondly, recv_vclock is used by recovery restart and must
-	 * always be valid.
+	 * received. Secondly, last received vclock is used by recovery restart
```

Comment:
By the way, we don't have `relay_restart_recovery` anymore, so you may shorten this comment.

Reply:
Thanks for noticing, updates some comments in a separate commit.